runelite-client/src/main/java/net/runelite/client/plugins/wiki/WikiPlugin.java

```diff
@@ -112,6 +126,11 @@ public void startUp()
 
 	@Override
 	public void shutDown()
+	{
+		removeWidgets();
+	}
+
+	private void removeWidgets()
```

Comment:
Remove the `clientThread` from here invocation and invoke it similarily to `addWidgets` with `clientThread.invokeLater(this::removeWidgets)